src/sailfishconnect.cpp

```diff
@@ -22,6 +22,9 @@
 #include <memory>
 #include <QLoggingCategory>
 #include <QQmlEngine>
+#include <QtQml>
```

Comment:
Why include QtQml?

Reply:
Build failed for me due to `qmlRegisterType` not being known to the compiler, at least with Qt headers as in latest SFOS SDK. Grepping showed that 
```
grep qmlRegisterType /host_targets/SailfishOS-2.1.4.13-i486/usr/include/qt5/ -R
/host_targets/SailfishOS-2.1.4.13-i486/usr/include/qt5/QtQml/qqml.h:int qmlRegisterType()
/host_targets/SailfishOS-2.1.4.13-i486/usr/include/qt5/QtQml/qqml.h:int Q_QML_EXPORT qmlRegisterTypeNotAvailable(const char *uri, int versionMajor, int versionMinor, const char *qmlName, const QString& message);
/host_targets/SailfishOS-2.1.4.13-i486/usr/include/qt5/QtQml/qqml.h:int qmlRegisterType(const char *uri, int versionMajor, int versionMinor, const char *qmlName)
/host_targets/SailfishOS-2.1.4.13-i486/usr/include/qt5/QtQml/qqml.h:int qmlRegisterType(const char *uri, int versionMajor, int versionMinor, const char *qmlName)
/host_targets/SailfishOS-2.1.4.13-i486/usr/include/qt5/QtQml/qqml.h:inline int qmlRegisterType(const QUrl &url, const char *uri, int versionMajor, int versionMinor, const char *qmlName)
/host_targets/SailfishOS-2.1.4.13-i486/usr/include/qt5/QtQml/qqml.h:        qWarning("qmlRegisterType requires absolute URLs.");
```
and <QtQML> seemed the only CamelCase forward header which pulled that in reliably. Despite what http://doc.qt.io/qt-5.6/qqmlengine.html#qmlRegisterType claims, seems that only in later versions `#include <QQmlEngine>` is enough.